third_party/terraform/utils/node_config.go.erb

```diff
@@ -250,6 +250,45 @@ func schemaNodeConfig() *schema.Schema {
 					Optional:   true,
 					ForceNew:   true,
 				},
+	<% end -%>
+	<% unless version == 'ga' -%>
+				"kubelet_config": {
+					Type:     schema.TypeList,
+					Optional: true,
+					MaxItems: 1,
+					Elem: &schema.Resource{
+						Schema: map[string]*schema.Schema{
```

Comment:
Can we force one of these to be required, or do an atleastoneof here? If not, can you add a comment explaining why?

Reply:
Made `cpu_manager_policy` `Required`